src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/SelectAttributeType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class SelectAttributeType
+ * @package Sylius\Bundle\AttributeBundle\Form\Type\AttributeType
+ */
+class SelectAttributeType extends AbstractType
```

Comment:
I'm not against this idea but before refactoring i prefere to wait other feedbacks.